src/api/app/controllers/build_controller.rb

```diff
@@ -1,4 +1,6 @@
 class BuildController < ApplicationController
+  include ApplicationHelper
```

Comment:
I don't like this at all. Helpers are for views, not to share controller code. ApplicationController is full of methods that can't be called as actions. What's so different about forward_from_backend?